network/proto/v1/network.proto

```diff
@@ -9,7 +9,7 @@ message Network {
     // Name of the network being indexed
```

Comment:
Is the v1 folder our way of versioning protos (schemas)?

Reply:
I think the aim is to version external schemas however I'd sayt that internal protos for data transfer between map and store methods don't need to follow this format. (I've got a commit that I'll push to the remote soon for handling non versioned binding generation that's still compatible with the versioned schemas)